test/Utilities/functions.jl

```diff
@@ -245,12 +245,12 @@ end
     @test g.terms == MOI.ScalarAffineTerm.([2, 6], [z, x])
     @test g.constant == 5
     h = it[[3, 1]]
-    @test h isa MOI.VectorAffineFunction
-    @test h.terms == MOI.VectorAffineTerm.([1, 1, 2, 2, 2], MOI.ScalarAffineTerm.([2, 6, 7, 1, 4], [z, x, y, z, x]))
+    @test h isa MOI.VectorAffineFunction    
+    @test sort(h.terms, by=t->t.output_index) == sort(MOI.VectorAffineTerm.([1, 1, 2, 2, 2], MOI.ScalarAffineTerm.([2, 6, 7, 1, 4], [z, x, y, z, x])), by=t->t.output_index)
```

Comment:
The RHS is already sorted isn't it ?

Reply:
yup, just pushed a commit removing the sort